config/default.yml

```
@@ -325,7 +325,7 @@ Performance/StringInclude:
   Enabled: 'pending'
   SafeAutoCorrect: false
   VersionAdded: '1.7'
-  VersionChanged: '1.12'
+  VersionChanged: '1.16'
```

Can you revert it? Because this update is not required.

```suggestion
  VersionChanged: '1.12'
```

---

Reverted.